testpar/testphdf5.c

```diff
@@ -402,9 +402,7 @@ main(int argc, char **argv)
     AddTest("split", test_split_comm_access, NULL, "dataset using split communicators", PARATESTFILE);
     AddTest("h5oflusherror", test_oflush, NULL, "H5Oflush failure", PARATESTFILE);
 
-#ifdef PB_OUT /* temporary: disable page buffering when parallel */
     AddTest("page_buffer", test_page_buffer_access, NULL, "page buffer usage in parallel", PARATESTFILE);
```

Comment:
Make sure that we're always testing page buffer access, even if that means that in parallel we're just making sure it's disabled.